scripts/install-aladdin.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ALADDIN_VERSION=${ALADDIN_VERSION:-$(\
+curl -s https://api.github.com/repos/fivestars-os/aladdin/releases/latest | \
+python3 -c 'import json,sys;print(json.load(sys.stdin)["tag_name"])' || \
```

Comment:
could also use `jq` - but since aladdin installs `jq` it seems reasonable to assume that `jq` might not always be available at install time, however we do expect python to always be available